be/src/exec/vectorized/cross_join_node.cpp

```diff
@@ -509,6 +524,55 @@ Status CrossJoinNode::_build(RuntimeState* state) {
     return Status::OK();
 }
 
+std::vector<bool> CrossJoinNode::could_rewrite_as_runtime_filter(
+        RuntimeState* state, const std::vector<ExprContext*>& ctxs,
+        const std::vector<RuntimeFilterBuildDescriptor*>& rf_descs) {
+    std::vector<bool> res(ctxs.size(), false);
+    for (int i = 0; i < rf_descs.size(); ++i) {
+        res[rf_descs[i]->build_expr_order()] = true;
+    }
+    return res;
+}
+
+StatusOr<std::list<ExprContext*>> CrossJoinNode::rewrite_runtime_filter(ObjectPool* pool,
+                                                                        const std::vector<bool>& could_rewrite,
+                                                                        Chunk* chunk,
```

Comment:
Are follwoing function helpful? 

```
    static JoinRuntimeFilter* create_runtime_bloom_filter(ObjectPool* pool, PrimitiveType type);
    static Status fill_runtime_bloom_filter(const ColumnPtr& column, PrimitiveType type, JoinRuntimeFilter* filter,
                                            size_t column_offset, bool eq_null);
```

Reply:
maybe a little hard, but I can move rewrite_runtime_filter to RuntimeFilterHelper

Reply:
It seems that JoinRuntimeFil is closer to Global RuntimeFilter, but currently cross join on RuntimeFilter is closer to Local RuntimeFilter (RuntimeInFilter)

Reply:
I move it to RuntimeFilterHelper